projects/1.21/assets/roost-ultimate/chicken_roost/lang/zh_cn.json

```
@@ -0,0 +1,470 @@
+  "entity.chicken_roost.c_apatite": "磷灰石鸡",
+  "entity.chicken_roost.c_apple": "苹果鸡",
+  "entity.chicken_roost.c_basalt": "玄武岩鸡",
+  "entity.chicken_roost.c_basalz": "岩石鸡",
+  "entity.chicken_roost.c_beetroot": "甜菜根鸡",
+  "entity.chicken_roost.c_biofuel": "生物燃料鸡",
+  "entity.chicken_roost.c_birchwood": "白桦鸡",
+  "entity.chicken_roost.c_bitumen": "沥青鸡",
+  "entity.chicken_roost.c_black": "黑鸡",
+  "entity.chicken_roost.c_blazepowder": "烈焰粉鸡",
+  "entity.chicken_roost.c_blazerod": "烈焰棒鸡",
+  "entity.chicken_roost.c_blitz": "狂风鸡",
+  "entity.chicken_roost.c_blizz": "暴雪鸡",
+  "entity.chicken_roost.c_blood": "血鸡",
+  "entity.chicken_roost.c_blutonium": "蓝钚鸡",
+  "entity.chicken_roost.c_bone": "骨白鸡",
+  "entity.chicken_roost.c_bonemeal": "骨粉鸡",
+  "entity.chicken_roost.c_brass": "黄铜鸡",
+  "entity.chicken_roost.c_bronze": "青铜鸡",
+  "entity.chicken_roost.c_brown": "棕鸡",
+  "entity.chicken_roost.c_carrot": "胡萝卜鸡",
+  "entity.chicken_roost.c_certusquartz": "赛特斯石英鸡",
+  "entity.chicken_roost.c_charcoal": "木炭鸡",
+  "entity.chicken_roost.c_chargedcertus": "充能赛特斯石英鸡",
+  "entity.chicken_roost.c_chorusfruit": "紫颂果鸡",
+  "entity.chicken_roost.c_chrome": "铬鸡",
+  "entity.chicken_roost.c_cinnabar": "朱砂鸡",
+  "entity.chicken_roost.c_clay": "黏土鸡",
+  "entity.chicken_roost.c_coal": "煤炭鸡",
+  "entity.chicken_roost.c_cobald": "钴鸡",
+  "entity.chicken_roost.c_cobble": "圆石鸡",
+  "entity.chicken_roost.c_coke": "焦炭鸡",
+  "entity.chicken_roost.c_constantan": "康铜鸡",
+  "entity.chicken_roost.c_copper": "铜鸡",
+  "entity.chicken_roost.c_crimstonstem": "绯红菌柄鸡",
+  "entity.chicken_roost.c_cyan": "青鸡",
+  "entity.chicken_roost.c_darkoak": "深色橡木鸡",
+  "entity.chicken_roost.c_diamond": "钻石鸡",
+  "entity.chicken_roost.c_diorite": "闪长岩鸡",
+  "entity.chicken_roost.c_electrum": "琥珀金鸡",
+  "entity.chicken_roost.c_elementium": "源质钢鸡",
+  "entity.chicken_roost.c_emerald": "绿宝石鸡",
+  "entity.chicken_roost.c_endereye": "末影之眼鸡",
+  "entity.chicken_roost.c_enderium": "末影鸡",
+  "entity.chicken_roost.c_enderpearl": "末影珍珠鸡",
+  "entity.chicken_roost.c_endstone": "末地石鸡",
+  "entity.chicken_roost.c_feather": "羽毛鸡",
+  "entity.chicken_roost.c_flint": "燧石鸡",
+  "entity.chicken_roost.c_fluixcrystal": "福鲁伊克斯鸡",
+  "entity.chicken_roost.c_ghasttear": "恶魂之泪鸡",
+  "entity.chicken_roost.c_ghostchicken": "Ghost Chicken - WIP",
+  "entity.chicken_roost.c_glass": "玻璃鸡",
+  "entity.chicken_roost.c_glowberries": "发光浆果鸡",
+  "entity.chicken_roost.c_glowstone": "荧石鸡",
+  "entity.chicken_roost.c_gold": "金鸡",
+  "entity.chicken_roost.c_granit": "花岗岩鸡",
+  "entity.chicken_roost.c_gravel": "沙砾鸡",
+  "entity.chicken_roost.c_gray": "灰鸡",
+  "entity.chicken_roost.c_green": "绿鸡",
+  "entity.chicken_roost.c_hepatizon": "黑色科林斯青铜鸡",
+  "entity.chicken_roost.c_honeycomb": "蜜脾鸡",
+  "entity.chicken_roost.c_ink": "墨囊黑鸡",
+  "entity.chicken_roost.c_invar": "因瓦合金鸡",
+  "entity.chicken_roost.c_iridium": "铱鸡",
+  "entity.chicken_roost.c_iron": "铁鸡",
+  "entity.chicken_roost.c_junglewood": "丛林木鸡",
```

```suggestion
  "entity.chicken_roost.c_junglewood": "丛林鸡",
```
和云杉这些统一格式，~~虽然我也不知道有啥格式（，橡木和深色橡木大概又要特殊了~~